testing/tests/Renovation/__meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "constellation": "viz",
```

Comment:
Let's make 'renovation' constellation

Reply:
In the `viz` constellation all tests work faster.